### ISSUE TITLE
[Bug]: `bottles-cli` still broken as of version 51.17

### ISSUE THREAD

**Issue by guihkx** [bottlesdevs/Bottles#3623]
### Describe the bug

This is similar to #3595, but not exactly the same.

`bottles-cli` is still not fully working as of v51.17:

```
$ flatpak --filesystem=$PWD run --command=bottles-cli com.usebottles.bottles run -b Default -e $PWD/rufus-4.6p.exe
15:14:48 (INFO) Forcing offline mode 
Traceback (most recent call last):
  File "/app/bin/bottles-cli", line 758, in <module>
    cli = CLI()
          ^^^^^
  File "/app/bin/bottles-cli", line 236, in __init__
    self.__process_args()
  File "/app/bin/bottles-cli", line 285, in __process_args
    self.run_program()
  File "/app/bin/bottles-cli", line 699, in run_program
    WineExecutor.run_program(bottle, program)
                                     ^^^^^^^
UnboundLocalError: cannot access local variable 'program' where it is not associated with a value
```

Downgrading to 51.15 makes it work again.

### To Reproduce

1\. Download [this test program](https://github.com/pbatard/rufus/releases/download/v4.6/rufus-4.6p.exe)
2\. Assuming you're using the Flatpak package, try to run the executable inside a bottle named `Default`:

```
flatpak --filesystem=$PWD run --command=bottles-cli com.usebottles.bottles run -b Default -e $PWD/rufus-4.6p.exe
```
3\. Observe the error message given:

```py
15:14:48 (INFO) Forcing offline mode 
Traceback (most recent call last):
  File "/app/bin/bottles-cli", line 758, in <module>
    cli = CLI()
          ^^^^^
  File "/app/bin/bottles-cli", line 236, in __init__
    self.__process_args()
  File "/app/bin/bottles-cli", line 285, in __process_args
    self.run_program()
  File "/app/bin/bottles-cli", line 699, in run_program
    WineExecutor.run_program(bottle, program)
                                     ^^^^^^^
UnboundLocalError: cannot access local variable 'program' where it is not associated with a value
```

### Package

Flatpak from Flathub

### Distribution

Arch Linux

### Debugging Information

```shell
N/A
```

**Comment by jntesteves** [comment]
Duplicate of #3599
This was fixed in #3605 but a new release hasn't been made yet.

**Comment by guihkx** [comment]
Thanks for the heads up! Closing then.
